src/xdocs/config_whitespace.xml

```diff
@@ -2967,6 +2971,20 @@ public void myTest() {
     for (;;){}               // OK
     for(;;){}                // violation, space after 'for' is required
 }
+
+public class TryStatement {
```

Comment:
Put under "for"

Examples like
`try(InputStream ignored = System.in) {}`

Reply:
Changes Done